GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+
+#### Removing a Maintainer
+
+Maintainers may resign at any time if they feel that they will not be able to
+continue fulfilling their project duties.
+
+Maintainers may also be removed after being inactive, failure to fulfill their
+Maintainer responsibilities, violating the Code of Conduct, or other reasons.
+Inactivity is defined as a period of very low or no activity in the project.
+A yearly maintainer ping, an e-mail sent to inactive maintainers, determines if the maintainer is still willing to fulfill their project duties.
+On failure to reply to the maintainer ping within the specified amount of time (usually a month), the maintainer will be removed.
+
+### Release Managers
+
+Release managers make sure the quarterly release comes out in time.
+They are one or more maintainers which were appointed for a specific release by the Virtual Maintainer Assembly (VMA).
+Their duties include setting the dates for feature freeze for the release, enforcing the feature freeze, coordinating the (mostly automated) tests of a release, writing the release notes and creating the tags defining the release and its release candidates.
+The full set of tasks can be found in the document [Managing a Release].
+Their duties end, once the release is out or if all bug-fixing point releases are out.
+
+### Admins
+
+GitHub admins are a special subgroup among RIOT maintainers.
+They are marked as such in the [maintainers list].
+They have more access rights to the RIOT repositories, such as granting access to a repository, adding new members to a team, or enabling protection for Git branches.
+Release managers might need to contact GitHub admins to configure the branch protection rules for the release branch.
+Beyond those technical duties and access rights, they do not have any special rights among maintainers.
+They are picked by the maintainers, usually based on seniority.
+The maintainers try to take care to spread the admin responsibility over several organizations within the maintainer body.
+
+There are also admins on the other RIOT discussion platforms, beyond technical administrative duties they do not have any special rights.
+These admins usually are appointed or self-appointed on merrit, i.e., whoever sets up the platform usually is (one of) its admin(s).
+
+### GitHub Owners
+Github owners are a special subgroup among RIOT GitHub admins.
+They are marked as such in the [maintainers list].
+Beyond this special status and the usual GitHub admin rules and duties they do not have any special rights among maintainers.
+
+### Moderators
+
+Moderators are responsible to enforce the values of the RIOT community within its discussion platforms. Each platform usually has its own set of moderators, a list of which can be found there.
+The forum moderators, e.g., can be found [here](https://forum.riot-os.org/g/moderators) (link requires you to be logged into the forum).
```

Comment:
We currently don't have any appointed moderators for other communication channels, do we? This text makes it sound as if we had (or at least would like to have) them, but doesn't give a hint on what those are.

Reply:
We do. The forum moderators where appointed by the community back when the forum was started.